src/model/cloud-runner/tests/cloud-runner-sync-environment.test.ts

```diff
@@ -0,0 +1,91 @@
+import { BuildParameters, ImageTag } from '../..';
+import CloudRunner from '../cloud-runner';
+import Input from '../../input';
+import { CloudRunnerStatics } from '../cloud-runner-statics';
+import { TaskParameterSerializer } from '../services/task-parameter-serializer';
+import UnityVersioning from '../../unity-versioning';
+import { Cli } from '../../cli/cli';
+import CloudRunnerLogger from '../services/cloud-runner-logger';
+import CloudRunnerOptions from '../cloud-runner-options';
+import GitHub from '../../github';
+import setups from './cloud-runner-suite.test';
+
+async function CreateParameters(overrides) {
+  if (overrides) {
+    Cli.options = overrides;
+  }
+  const originalValue = GitHub.githubInputEnabled;
+  GitHub.githubInputEnabled = false;
+  const results = await BuildParameters.create();
+  GitHub.githubInputEnabled = originalValue;
+  delete Cli.options;
+
+  return results;
+}
```

Comment:
```suggestion
async function CreateParameters(overrides) {
  if (overrides) Cli.options = overrides;

  return BuildParameters.create();
}
```

Reply:
I think this can be simplified, because setup is always called (and always intended to be) before this method

Reply:
🤔